asset/css/page-nav.css

```diff
@@ -0,0 +1,51 @@
+/* Page navigation */
+
+#page-nav {
+    border-left: solid 1px lightgrey;
+    display: inline-block;
+    direction: rtl;
+    max-height: 80vh;
+    overflow: auto;
+    position: fixed;
+    right: 0;
+    transition: 0.4s;
+    width: 300px;
+    -webkit-transition: 0.4s;
+}
+
+#page-nav a:link, #page-nav a:visited, #page-nav a:hover{
+    color: black;
+    text-decoration: none;
+}
+
+#page-nav a:hover{
+    color: #0076FF;
+}
+
+#page-nav a.active {
+    background-color: transparent;
+    border-radius: 0;
+    border-right: 2px solid #0076FF;
+}
+
+#page-nav-content-wrapper {
+    margin-right: 300px;
+    transition: 0.4s;
+    -webkit-transition: 0.4s;
+}
+
+/* Responsive site navigation */
+
+/* Hides page navigation on screen widths smaller than 1600px */
+@media screen and (max-width: 1599.98px) {
```

Comment:
I switched to my laptop for the first time in reviewing this PR, and the page navigation didn't appear, even though my resolution is 1920x1080.

I can't seem to figure out the exact problem, will let you know once I have further time to dive into this PR.

Reply:
If you zoom in within the browser, `pageNav` will disappear as well.

Also, within Windows, if your UI scaling is more than 100%, browsers will treat your resolution as being smaller than actual.

Reply:
> Also, within Windows, if your UI scaling is more than 100%, browsers will treat your resolution as being smaller than actual.

I see, that could be the problem. I wonder if there's any way around it, because it feels like a pity to miss out this feature when there's space for it (I forced it to always render and it seems to have sufficient space):

![pagenav](https://user-images.githubusercontent.com/3168908/50959398-d9827e00-14fd-11e9-955b-82c2f0d71ccf.png)

Reply:
My original intention was to give the contents a minimum of 1000px width for the `<body>` content. (`siteNav` and `pageNav` both are `300px`)

But upon checking the `siteNav.CSS` which uses `(max-width: 991.98px)`, it might be more appropriate to use `(max-width: 1199.98px)[Bootstrap Large responsive breakpoint]` + `100px` to accommodate the `300px` siteNav. Thus leaving the `<body>` content with a minimum of `692px` width.

Reply:
> I don't see a way around it since its something we can't control. We can only make it slightly better by implementing the solution above (dropping `1599.98px` to `1299.98px`).

I see. Let's try the solution then. I am not very sure about the common resolutions and scalings that students have, but if multiple students are facing the same problem and their screen space is reasonably big, then we may need to think of some other solutions as well.